### PR TITLE
Update README.md for Ember-CLI 0.2.3

### PR DESCRIPTION
**Pull request by al3x-edge**
Ember CLI updated it's install command.  This commit puts documentation inline with the new command while keeping the old syntax around for those who need it.

### CODE REVIEW

**Comment by poteto** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -8,8 +8,13 @@
 This `ember-cli` addon adds a simple flash message service to your app. It's injected into all `Controllers`, `Routes`, `Views` and `Components` by default ([you can change this](#service-defaults)), or lazily injected with `Ember.inject.service`.
 
 ## Installation
-You can install either with `ember install:addon`:
+You can install either with `ember install`:
 
+For Ember CLI >= 0.2.3:
+```shell
```
Comment (poteto) [inline README.md]: Could you please add a newline before and after the code blocks? It renders the same, but I'd like to keep the formatting consistent

Comment (al3x-edge) [inline README.md]: Of course.

**Comment by al3x-edge** [comment]
@poteto -- Looks good to go on tests and squashed the commit to just the original and linebreaks.

**Comment by poteto** [comment]
Awesome, thank you! :)

**Comment by al3x-edge** [comment]
Thanks Lauren =]

**Pull request merged**